lib/rules/no-extra-parens.js

```diff
@@ -927,7 +955,8 @@ module.exports = {
             LogicalExpression: checkBinaryLogical,
 
             MemberExpression(node) {
-                const nodeObjHasExcessParens = hasExcessParens(node.object);
+                const shouldAllowWrapOnce = isMemberExpInNewCallee(node) && hasCallExpInMemberExp(node);
+                const nodeObjHasExcessParens = shouldAllowWrapOnce ? hasDoubleExcessParens(node.object) : hasExcessParens(node.object);
 
                 if (
                     nodeObjHasExcessParens &&
```

Comment:
I think this flow might miss some extra double parens:

```js
((a + b)).length;
```

Reply:
@mdjermanovic 
Yes it seems a false positive in current(v7.2.0) "no-extra-parens" - [online-demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLWV4dHJhLXBhcmVuczogXCJlcnJvclwiICovXG5cbigoYSArIGIpKS5sZW5ndGg7Iiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMSwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6e30sImVudiI6e319fQ==)
I'll try to fix it:) Thanks